runtime/vm/jvminit.c

```diff
@@ -3954,6 +3954,7 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		}
 	}
 
+	vm->checkpointState.criuRestoreStartTimeInNanoseconds = 0;
```

Comment:
We dont need a two of these, only one will be used by the API. It should still be negative if we havent done a restore

Reply:
Since the original native `getLastRestoreTimeImpl` could be used for debugging purposes https://github.com/eclipse-openj9/openj9/pull/18680#discussion_r1454119453, I kept both `lastRestoreTimeMillis` and `criuRestoreStartTimeInNanoseconds` in `vm->checkpointState`.